ipu-plugin/pkg/ipuplugin/vendorservice.go

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) 2024 Intel Corporation.  All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipuplugin
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/openshift/dpu-operator/api/dpu_api"
+	"github.com/vishvananda/netlink"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type VendorManagementServer struct {
+	dpu_api.UnimplementedVendorManagementServiceServer
+	ip   string
+	port int
+	mode string
+}
+
+func NewVendorManagementService(ip string, port int, mode string) *VendorManagementServer {
+	return &VendorManagementServer{
+		ip:   ip,
+		port: port,
+		mode: mode,
+	}
+}
+
+const (
+	HOST_VPORT_ID = "03"
```

Comment:
If these constants are not supposed to be exported then we should name them using smallcap for starting letter and CamelCase is standard go-convention we should follow here too.

Reply:
Done